subsys/net/ip/net_core.c

```diff
@@ -71,39 +71,39 @@ static inline enum net_verdict process_data(struct net_pkt *pkt)
 	if (!net_pkt_is_raw_processed(pkt)) {
 		net_pkt_set_raw_processed(pkt, true);
 		net_packet_socket_input(pkt, ETH_P_ALL, SOCK_RAW);
+		return NET_CONTINUE;
```

Comment:
I don't think packet-socket processing would require to recalculate the packet priority?